tests/e2e/e2e.sh

```diff
@@ -17,69 +17,25 @@
 set -eux
 
 export ARTIFACTS="${ARTIFACTS:-$(mktemp -d)}"
-
-function cleanup_kind_cluster() {
-    kind export logs --name istio-testing "${ARTIFACTS}/kind"
-    kind delete cluster --name=istio-testing
-}
-
-function setup_kind_cluster() {
-  # Delete any previous e2e KinD cluster
-  echo "Deleting previous KinD cluster with name=istio-testing"
-  if ! (kind delete cluster --name=istio-testing) > /dev/null; then
-    echo "No existing kind cluster with name istio-testing. Continue..."
-  fi
-
-  trap cleanup_kind_cluster EXIT
-
-  # Create KinD cluster
-  if ! (kind create cluster --name=istio-testing --loglevel debug --retain --wait 30s); then
-    echo "Could not setup KinD environment. Something wrong with KinD setup. Exporting logs."
-    exit 1
-  fi
-
-  KUBECONFIG="$(kind get kubeconfig-path --name="istio-testing")"
-  export KUBECONFIG
-
-cat <<EOF > metallb-config.yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  namespace: metallb-system
-  name: config
-data:
-  config: |
-    address-pools:
-    - name: default
-      protocol: layer2
-      addresses:
-      - 172.17.255.1-172.17.255.250
-EOF
-
-  kubectl apply -f https://raw.githubusercontent.com/google/metallb/v0.8.1/manifests/metallb.yaml
-  kubectl apply -f ./metallb-config.yaml
-}
+export GOPATH=$(cd ../../../; pwd)
 
 function setup_docker() {
-  HUB=istio-testing TAG=latest make -f Makefile.core.mk controller docker
-  kind --loglevel debug --name istio-testing load docker-image istio-testing/operator:istio-testing
+  HUB=gcr.io/istio-testing TAG=1.5-dev make -f Makefile.core.mk controller docker
+  kind --loglevel debug --name e2e load docker-image gcr.io/istio-testing/operator:1.5-dev
 }
 
-setup_kind_cluster
-setup_docker
-
 mkdir -p "${ARTIFACTS}/out"
 
 ISTIO_DIR="${GOPATH}/src/istio.io/istio"
 
 # Create a clone of the Istio repository
 if [[ ! -d "${ISTIO_DIR}" ]]
 then
-  git clone https://github.com/sdake/istio.git "${ISTIO_DIR}"
+  git clone https://github.com/istio/istio.git "${ISTIO_DIR}"
```

Comment:
surprised this merged with sdake as my repo.  I was using that for testing only and have since overwritten my master.  github.com/istio/istio.git contains all of the changes needed for this PR to work - so this line looks correct.

Reply:
@istio/wg-environments-maintainers please take care when reviewing PRs to not **approve** PRs that contain personal git repositories.  Thewse personal git repositories may be used for testing if they are from trusted developers, however, they should not be merged and testing should not be permitted for folks new to the project (secrets can be exposed...)

Cheers
-steve